src/app/components/wrapper-components/part-creation/part-creation.component.ts

```diff
@@ -347,7 +347,7 @@ export class PartCreationComponent implements OnInit, OnDestroy {
             for (const user of foundUsers) {
                 if (user.data.state === 'offline') {
                     await this.removeUserFromLobby(Utils.getNonNullable(user.data.username));
-                    await ErrorLoggerService.logError('PartCreationComponent', 'user is already offline', { username: user.data.username, userId: user.id });
+                    ErrorLoggerService.logError('PartCreationComponent', 'user is already offline', { username: user.data.username, userId: user.id });
```

Comment:
.getMinimalUser ?

Reply:
Et si on créeait une méthode static MinimalUser.from(document<user>) ce serait JUSTE pour ici je suppose, et donc Over Kill (mais paradoxalement pas OK, du coup, hoho) ?

Reply:
Fait, mais je trouve que ça perd en lisibilité ici ! Surtout pour de l'info de debug.

Reply:
Comme mentionné, si c'était pour un seul usage, c'est overkill, je demandais pas de le faire forcément :p Surtout si faut utiliser le "// allow me to redeclare please" ^^

Reply:
c'était deux usages ! Et faut pas proposer alors :D Du coup, je revert ?

Reply:
Demander "si c'était rose ce serait bien ou pas?" ne veut pas dire "vazy repeint MENAN toute la maison en rose" :D 
Si tu trouvait ça mieux avant tu peux revert :p

Reply:
Ce commentaire si c'était "si c'était rose ça serait peut-être bien, même si on l'utilise qu'une fois", et le commentaire qui suivait directement allait dans le sens "ah bah on a encore besoin du rose, partons là dessus" ! :D Je repasserai peut-être dessus au prochain commit après relecture.

Reply:
"Et si on créeait *du rose* ce serait JUSTE pour ici je suppose, et donc Over Kill (mais paradoxalement pas OK, du coup, hoho) ?" ce à quoi tu répond "fait"
Alors que bon, je dit même dans le message que c'est pas bien je crois haha :D !

Reply:
Mais y'avait un autre message qui disait autre chose, qui depuis a été résolu ! Du coup enlevé !